test/connectors/PenaltiesConnectorSpec.scala

```diff
@@ -211,45 +220,45 @@ class PenaltiesConnectorSpec extends SpecBase {
   }
 
   "getMultiplePenaltiesForPrincipleCharge" should {
-    s"return $Some $JsValue when the connector call succeeds" in new Setup {
-      when(mockHttpClient.GET[HttpResponse](any(), any(), any())(any(), any(), any()))
-        .thenReturn(Future.successful(HttpResponse(Status.OK, multiplePenaltiesJson.toString())))
+    s"return Right with the correct model when the connector call succeeds" in new Setup {
+      when(mockHttpClient.GET[MultiplePenaltiesResponse](any(), any(), any())(any(), any(), any()))
+        .thenReturn(Future.successful(Right(multiplePenaltiesModel)))
       when(mockAppConfig.multiplePenaltyDataUrl(any(), any()))
         .thenReturn("http://url/url")
 
-      val result: Option[JsValue] = await(connector.getMultiplePenaltiesForPrincipleCharge("1234", "123456789"))
-      result.isDefined shouldBe true
-      result.get shouldBe multiplePenaltiesJson
+      val result: MultiplePenaltiesResponse = await(connector.getMultiplePenaltiesForPrincipleCharge("1234", "123456789"))
+      result.isRight shouldBe true
+      result shouldBe Right(multiplePenaltiesModel)
     }
 
     s"return $None when $NO_CONTENT is returned from the call" in new Setup {
-      when(mockHttpClient.GET[HttpResponse](any(), any(), any())(any(), any(), any()))
-        .thenReturn(Future.successful(HttpResponse(Status.NO_CONTENT, "")))
+      when(mockHttpClient.GET[MultiplePenaltiesResponse](any(), any(), any())(any(), any(), any()))
+        .thenReturn(Future.successful(Left(NoContent)))
       when(mockAppConfig.multiplePenaltyDataUrl(any(), any()))
         .thenReturn("http://url/url")
 
-      val result: Option[JsValue] = await(connector.getMultiplePenaltiesForPrincipleCharge("1234", "123456789"))
-      result.isDefined shouldBe false
+      val result: MultiplePenaltiesResponse = await(connector.getMultiplePenaltiesForPrincipleCharge("1234", "123456789"))
+      result.isRight shouldBe false
```

Comment:
You can use isLeft and it may be worth adding `result.left.get shouldBe Left(NoContent)`